### DISCUSSION TITLE
add supprort for snowpack for speed

### DISCUSSION THREAD

**Discussion by MohdAhmad1** [vercel/next.js#24045] [Ideas]
**Describe the feature you'd like to request**

next js compilation is slow so we want fast compilation

**Describe the solution you'd like**

please add support of snowpack over webpack because snowpack is very fast.

**Describe alternatives you've considered**

snowpack

**Comment by odc-john** [comment]
https://www.snowpack.dev/guides/server-side-render will solve dev friendliness problem. i was about to just create more own ssr framework if nextjs doesn't do this. debugging with breakpoints nextjs large scale apps as well as per page ssr compilation is atrocious and cause the debugger to digitally hemorrhage.

snowpack is the way
